index.html

```diff
@@ -78,6 +88,7 @@ <h1 class="section-titlef">Facebook 360</h1>
         <section class="section2">
             <div class="right-block">
                 <img class="image-section2" src=".images/Snapshoot Portfolio1.png" alt="project image">
+                <img class="img-sec2" src=".images/Snapshoot Portfolio section2.png" alt="project image">
             </div>
             <div class="left-block">
                 <div class="experience1">
```

Comment:
- [ ] As mentioned by the previous reviewer https://github.com/Kidd254/Lawrence-s-Portfolio/pull/4#discussion_r1043838933 Please fix the project cards to match the figma design, It should be alternating between texts and image, for example if the 1st project card's image is on left and project description is on the right, the 2nd project's cards image will be on the right and description on the left, currently all your cards have images on the same side and description on the same side too. pleas see :point_down: 
![image](https://user-images.githubusercontent.com/97521557/206868068-f8e21d62-aa66-45d3-8429-edb4a6371378.png)

how it should be :point_down: 
![image](https://user-images.githubusercontent.com/97521557/206868118-6012db3e-1004-4cf2-a6fb-37b5cfa77380.png)